src/meta/src/stream/meta.rs

```diff
@@ -133,6 +137,7 @@ where
                 table_fragment.table_id()
             )))),
             Entry::Vacant(v) => {
+                table_fragment.seq = 1;
```

Comment:
Ditto. We could just set `seq` as 1 in `new` instead of 0.

Reply:
fixed